Nc-News/src/api.js

```diff
@@ -4,8 +4,9 @@ const api = axios.create({
   baseURL: "https://news-server-api.onrender.com/api",
 });
 
-export function getArticles() {
-  return api.get("/articles");
+export function getArticles(params) {
+  console.log(params);
+  return api.get("/articles", params);
```

Comment:
Love that you've refactored this function to take in a params object rather than make a whole new function 👍